src/app/FeedbackResults.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// import { NavLink } from 'react-router-dom';
```

Comment:
please delete comments before merging to master. espacially imports you dont need